app-crypt/efitools/efitools-1.8.1.ebuild

```diff
@@ -25,7 +25,8 @@ DEPEND="${RDEPEND}
 	sys-boot/gnu-efi
 	virtual/pkgconfig"
 
-PATCHES=( "${FILESDIR}/${P}-libressl-compatibility.patch" )
+PATCHES=( "${FILESDIR}/1.7.0-Make.rules.patch"
```

Comment:
once more than one item is added to PATCHES, you'll want to unwrap it

```
PATCHES=(
    ...
    ...
)
```

Reply:
Done in the updated branch.